README.md

```diff
@@ -51,6 +51,16 @@ genius-lyrics-api exposes the following methods:
 Automatically get the lyrics from genius.com using the `title` & `artist` fields.<br/>
 Returns a promise that resolves to a string containing lyrics. Returns `null` if no lyrics are found.
 
+### `getAlbumArt(options)`
+
+Automatically get the albumArt from genius.com using the `title` & `artist` fields.<br/>
+Returns a promise that resolves to a string containing a url. Returns `null` if no lyrics are found.
+
+### `getSong(options)`
+
+Automatically gets both the lyrics and albumArt from genius.com using the `title` & `artist` fields.<br/>
+Returns a promise that resolves to a array containing the lyrics and the album art url in the format of `[lyrics, url]`. Array element equals `null` if no lyrics or urls are found.
```

Comment:
Wouldn't it be better to return an object with `lyrics` & `albumArt` properties, rather than an array? Just a thought